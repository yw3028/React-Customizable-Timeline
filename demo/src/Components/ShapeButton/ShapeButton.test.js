import { cleanup, render, screen } from '@testing-library/react';
import React from 'react';
import ShapeButton from './ShapeButton';

const renderComponent = shapes => render(<ShapeButton shapes={shapes} />);

describe('ShapeButton Component', () => {
  afterEach(() => {
    cleanup();
  });

  test('renders correctly', async () => {
    const component = renderComponent(true);
    expect(component.baseElement).toMatchSnapshot();
  });
});
