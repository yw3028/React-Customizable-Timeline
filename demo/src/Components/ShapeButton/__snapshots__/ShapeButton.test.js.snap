// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShapeButton Component renders correctly 1`] = `
<body>
  <div />
</body>
`;
